frontend/src/graph/ui/Chart.vue

```diff
@@ -16,8 +18,16 @@
 
     // https://github.com/taye/interact.js
     import interact from 'interactjs';
-    import cytoscape from 'cytoscape';
-    import dagre from 'cytoscape-dagre';
+
+    // for cytoscape
```

Comment:
Since we will remove the cytoscape from the package.json, we can also remove this part.